packages/core/src/Cardano/types/Generic.ts

```diff
@@ -0,0 +1,4 @@
+export interface Fraction {
```

Comment:
`Generic` is an overloaded term in TypeScript when referring to types, so probably best to rename this module

Reply:
Fixed in `08bf2b3ab0171306eeec830c97d025c9f6e1e057` called `UtilityTypes` unless there is a more descriptive name preferred